lib/cocoapods/config.rb

```diff
@@ -28,6 +28,11 @@ class Config
     # Applies the given changes to the config for the duration of the given
     # block.
     #
+    # @param [Hash<#to_sym,Object>] changes
+    #        the changes to merge temporarily with the current config
+    #
+    # @yield [] is called while the changes are applied
```

Comment:
There are no arguments passed to the block. I thought that way, it's more explicit.